tc/sgx/trusted_worker_manager/enclave/base.edl

```diff
@@ -32,5 +32,14 @@ enclave {
         void ocall_Log(int level, [in, string] const char* str);
         void ocall_SetErrorMessage([in, string] const char* msg);
         void ocall_GetTimer([out] uint64_t* value);
+         uint32_t ocall_Process(uint32_t handlerId,
```

Comment:
Can we move thsi function to a separate edl file instead of adding to base.edl file

Reply:
Done